ibis/backends/sql/datatypes.py

```diff
@@ -1038,6 +1038,16 @@ class FlinkType(SqlglotType):
     default_decimal_precision = 38
     default_decimal_scale = 18
 
+    @classmethod
+    def from_ibis(cls, dtype: dt.DataType) -> sge.DataType:
+        """Convert a sqlglot type to an ibis type."""
+
+        typ = super().from_ibis(dtype)
+        if not dtype.nullable:
+            typ.set("kind", sge.NotNullColumnConstraint())
```

Comment:
`NotNullColumnConstraint()` isn't a valid type of value for `kind`.

Can you write down the thing you're trying to do and we can work towards a solution?